libs/coin-modules/coin-tron/src/bridge/synchronization.integ.test.ts

```diff
@@ -84,23 +126,28 @@ describe("Sync Accounts", () => {
     expect(account.tronResources).toEqual(defaultTronResources);
   });
 
-  test.each([
-    "TL24LCps5FKwp3PoU1MvrYrwhi5LU1tHre",
-    "TAVrrARNdnjHgCGMQYeQV7hv4PSu7mVsMj",
-    "THAe4BNVxp293qgyQEqXEkHMpPcqtG73bi",
-    "TRqkRnAj6ceJFYAn2p1eE7aWrgBBwtdhS9",
-    "TUxd6v64YTWkfpFpNDdtgc5Ps4SfGxwizT",
-    "TY2ksFgpvb82TgGPwUSa7iseqPW5weYQyh",
+  it.each([
+    {
+      id: "TL24LCps5FKwp3PoU1MvrYrwhi5LU1tHre",
+      expectedAccount: JSON.parse(JSON.stringify(account1Fixture), reviver),
```

Comment:
I think we should compare parts of the account (part of history for example, address, etc), anything that is stable

Reply:
Ok, I will break down with some reliable tests.